tags/msa.tag

```diff
@@ -0,0 +1,5 @@
+type: text
+
+---
+
+Since December 1st, all new Minecraft accounts are now using the new Microsoft Account system, which currently isn't supported by Geyser. We're actively working on a fix, please stay patient while we get the system implemented. We'll likely make a GeyserNews announcement when the fix is released, so if you'd like to be notified, you can run `!rank GeyserNews` in <#613194762249437245>.
```

Comment:
We likely will not make a Geyser News announcement if the fix is anytime soon; I would leave that part out.

Reply:
Done.